apps/auracast/src/main.c

```diff
@@ -60,12 +63,12 @@ static uint16_t bis_handles[MYNEWT_VAL(BROADCASTER_CHAN_NUM)];
 static struct os_callout audio_broadcast_callout;
 
 static int audio_data_offset;
+static uint8_t auracast_adv_instance;
+
+#if STOP_ON_BUTTON_PRESS
```

Comment:
this can be 
#if  MYNEWT_VAL(BROADCASTER_STOP_BUTTON) >= 0

so no need for extra wrap macro (this is how we typically do this in codebase)

Reply:
@MariuszSkamra could you help with this?